app/notifications/notifications.component.ts

```diff
@@ -27,8 +31,18 @@ export class NotificationsComponent implements OnInit {
       .catch(
         (error: any) => {
           this.notificationsResponseHandled = true;
-          this.toasterService.pop( 'error', 'Error', 'Not able to fetch your notifications.');
+          this.getTranslatedData = this.translatedData('Not able to fetch your notifications');
```

Comment:
Having a local variable for the translated string should be enough, no declaration in the class needed.